src/components/Main/Main.html

```diff
@@ -0,0 +1,35 @@
+<div class="main">
+  <aside class="main__sidebar mdc-drawer mdc-drawer--dismissible">
+    <div class="mdc-drawer__content">
+      <div class="mdc-list">
+        <a class="mdc-list-item mdc-list-item--activated" href="#" aria-selected="true">
+          <i class="material-icons mdc-list-item__graphic" aria-hidden="true">library_music</i>
+          <span class="mdc-list-item__text mdc-typography--headline4">My Songs</span>
+        </a>
+        <a class="mdc-list-item" href="#">
+          <i class="material-icons mdc-list-item__graphic" aria-hidden="true">album</i>
+          <span class="mdc-list-item__text mdc-typography--headline4">Albums</span>
+        </a>
+        <a class="mdc-list-item" href="#">
+          <i class="material-icons mdc-list-item__graphic" aria-hidden="true">person_outline</i>
+          <span class="mdc-list-item__text mdc-typography--headline4">Artists</span>
+        </a>
+        <a class="mdc-list-item" href="#">
+          <i class="material-icons mdc-list-item__graphic" aria-hidden="true">info</i>
+          <span class="mdc-list-item__text mdc-typography--headline4">About</span>
+        </a>
+      </div>
+    </div>
+  </aside>
+  <div class="main__wrapper mdc-drawer-app-content">
+    <header class="main__header mdc-top-app-bar app-bar"></header>
+    <main class="main__section main-content">
+      <div class="main__content-mount mdc-top-app-bar--fixed-adjust">
+        <h5 class="mdc-typography--headline3">Albums</h5>
+      </div>
+      <div class="main__share-view"></div>
```

Comment:
there should no be specific mount point for share view. It should be mounted into common mount point where albums, artists and about view mount

Reply:
probably inside `main__content-mount`